osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -48,6 +48,9 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
 
             double sliderFactor = aimRating > 0 ? aimRatingNoSliders / aimRating : 1;
 
+            double aimDifficultyStrainCount = ((OsuStrainSkill)skills[0]).CountDifficultStrains();
+            double speedDifficultyStrainCount = ((OsuStrainSkill)skills[2]).CountDifficultStrains();
```

Comment:
At some point I'd really like to rid of the indexing of skills because it makes me anxious but future problem..